File: src/Tokyo.Service/Models/Core/TimeNormTagMapping.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Tokyo.Service.Models.Core
{
    public class TimeNormTagMapping
    {
        public int TimeNormTagMappingId { get; set; }
        public int TimeNormId { get; set; }
        public TimeNorm TimeNorm { get; set; }
        public int TimeNormTagId { get; set; }
        public TimeNormTag TimeNormTag { get; set; }
    }
}

File: src/Tokyo.Client/Typewriter/TimeNormTagMapping.ts


import {TimeNorm} from "./TimeNorm";
import {TimeNormTag} from "./TimeNormTag"; 
export class TimeNormTagMapping { 
    TimeNormTagMappingId: number;
    TimeNormId: number;
    TimeNorm: TimeNorm | undefined;
    TimeNormTagId: number;
    TimeNormTag: TimeNormTag | undefined;
}